package.json

```diff
@@ -15,15 +15,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@bigcommerce/big-design": "^0.27.0",
+    "@bigcommerce/big-design": "^0.30.2",
     "firebase": "^8.2.6",
     "jsonwebtoken": "^8.5.1",
     "mysql": "^2.18.1",
-    "next": "^10.2.3",
+    "next": "^11.1.1",
```

Comment:
Was there a reason for not bumping this to the latest? (`^12.0.4`)?

Reply:
yeah, 12+ has major breaking changes - to go down that route should be a separate PR - bumping to 11 doesn't include breaking changes and fixes the security vulnerability